integration-tests/tests/preview-dataset.js

```diff
@@ -0,0 +1,74 @@
+const _ = require('lodash/fp')
+const { signIntoTerra, click, clickable, clickTableCell, findElement, findText, findTableCellText, getTableCellPath, getTableHeaderPath, input, waitForNoSpinners } = require('../utils/integration-utils')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { withUserToken } = require('../utils/terra-sa-utils')
+const { dismissNotifications } = require('../utils/integration-utils')
+
+const datasetName = 'Cell hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics'
+
+const assertEquals = (expected, actual) => {
+  if (expected !== actual) {
+    throw new Error("Expected value " + expected + " does not equal actual " + actual)
```

Comment:
We use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) when we want to substitute values (like Python's f-strings).
```suggestion
    throw new Error(`Expected value ${expected} does not equal actual ${actual}`)
```

Reply:
I ended up removing this function, but that is good to know!

Reply:
👍🏼